### DISCUSSION TITLE
Now This is a Mac OS Setup

### DISCUSSION THREAD

**Discussion by jrock2004** [nicknisi/dotfiles#229] [Ideas]
So I found a really cool video from [Josean Martinez](https://www.youtube.com/@joseanmartinez) that showed me something that I am excited that I can do on my Mac. Thought I would share just in case someone here wants something similar.

![image](https://github.com/user-attachments/assets/5c914f52-d171-4eb8-98bb-53cf0dda819a)

![image](https://github.com/user-attachments/assets/da4ae469-17a3-400c-b361-3ebee58cfe61)

**Yabai:** Tiling manager
**SKHD:** Use keyboard to navigate around
**Sketchybar:** Top of screen menu

**Comment by nicknisi** [comment]
This looks super interesting! Thanks for sharing! I did use skhd and yabai for a bit, but when I switched to two monitors I ended up feeling like it got in the way more, but maybe I should give it another try. I'd also never heard of Sketchybar but that looks really cool and something I always find attractive in linux setups.

Thanks again for sharing!

**Reply by jrock2004** [reply]
Yeah I found it in a post and was like woah. I don’t use 2 monitors so not sure if support has improved.
